inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -1378,7 +1378,13 @@ void prepare_conv_eltw_read_write_opt::run(program_impl& p) {
     while (itr != fused_conv_eltw_nodes.end()) {
         auto node_itr = itr++;
         auto& node = (*node_itr);
+        bool optimizeRW = true;
```

Comment:
This part requires some meaningful description

Reply:
Added comments